flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1309,10 +1309,10 @@ private ApplicationReport startAppMaster(
                                     + appId);
                     // break ..
                 case RUNNING:
-                    LOG.info("YARN application has been deployed successfully.");
+                    LOG.info("YARN application {} has been deployed successfully.", appId);
```

Comment:
In my experiences using yarn + flink, this enrichment is not needed because the application id is just around.
So I think these changes to `YarnClusterDescriptor` do not seem to be necessary and can be omitted.